pom.xml

```diff
@@ -26,7 +26,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <dropwizard.version>2.1.12</dropwizard.version>
+        <dropwizard.version>2.1.2</dropwizard.version>
```

Comment:
We updated to `2.1.12` before as it was the last supported version for dropwizard `2.x.x` but it ended up not running/discovering our tests so we reverted back.